stock_move_automatic_assign/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# (c) 2017 Daniel Campos <danielcampos@avanzosc.es> - Avanzosc S.L.
```

Comment:
En realidad hay que poner Copyright

Visto en OCA: the `(c)` or  `©` has to be accompanied by `Copyright` in order to be valid